docs/buildah-run.md

```diff
@@ -156,6 +156,12 @@ whenever the rootfs is on top of a ramdisk.
 Note: You can make this option the default by setting the BUILDAH\_NOPIVOT
 environment variable.  `export BUILDAH_NOPIVOT=true`
 
+**--security-opt** options
```

Comment:
This is totally unimplemented, it looks like this option does nothing.  I think it would be better to remove the option altogether and wait for users to complain.  Or open an issue to actually implement it.

Reply:
:embarrassed emoji: If I had taken the time to do more than a simple `grep`, I would've detected this.

I have removed the option and will push that once the `--userns-[gu]id-map` question is resolved. Thank you.